sdk/cosmos/azure_data_cosmos/src/models/throughput_properties.rs

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+use std::borrow::Cow;
+
+use azure_core::{
+    headers::{AsHeaders, HeaderName, HeaderValue},
+    Model,
+};
+use serde::{Deserialize, Serialize};
+
+use crate::{constants, models::SystemProperties};
+
+const OFFER_VERSION_2: &str = "V2";
+
+#[derive(Model, Clone, Default, Debug, Deserialize, Serialize)]
```

Comment:
Models should not derive `Debug` or implement `Display`, in most cases. New in the guidelines. Too easy to leak PII.

Reply:
Yep, I remember that now. I had implemented Debug during testing but am happy to remove it. I should say, we know this is all non-PII data, but I'm also fine with the default of not implementing `Debug` unless we have a specific reason to do so.

Reply:
I need to go through and change how our "testing" example works before I can do this everywhere, but I'll remove it from throughput_properties here. The example uses `Debug` quite a bit and I'd like to done something more consistent there.